core/templates/google-analytics.initializer.ts

```diff
@@ -18,14 +18,23 @@
 
 import constants from 'assets/constants';
 
+declare var GA_ANALYTICS_ID: string;
+declare var UA_ANALYTICS_ID: string;
+
+initializeGoogleAnalytics();
+
 export function initializeGoogleAnalytics() {
   if (!constants.CAN_SEND_ANALYTICS_EVENTS) {
     // Mock gtag function will prevent sending analytics to google.
     window.gtag = function() {}
     return;
   }
 
-  if (constants.ANALYTICS_ID && constants.SITE_NAME_FOR_ANALYTICS) {
+  if (
+    GA_ANALYTICS_ID &&
+    UA_ANALYTICS_ID &&
+    constants.SITE_NAME_FOR_ANALYTICS
```

Comment:
Is it better to move this into webpack too so that all the analytics constants are all in one place?

Not sure, just checking.